docs/tutorials/tutorial-0.rst

```diff
@@ -75,7 +77,7 @@ There is a new folder in your project called 'iOS', which contains the Xcode pro
 
 .. code-block:: bash
 
-    open iOS/Tutorial\ 0.xcodeproj
+    open iOS/Tutorial\ 0.xcodeproj OR ./linux/tutorial_0
 
 You can test the app by running it in Xcode. As our application only shows a message, the iOS application will show only a blank screen. You can check if it is working in the console logs, which should contain something like this:
```

Comment:
These instructions will need to be updated as well - no point talking about XCode if you're on linux.

In fact, it might be better to write a completely separate "getting started on Linux" tutorial, and give the user the option of which tutorial they run based on the platform they're using.